requirements.txt

```diff
@@ -4,7 +4,7 @@ sqlalchemy==1.2.12
 psycopg2==2.7.5
 csvkit==1.0.3
 
-https://github.com/opencivicdata/python-opencivicdata/zipball/longer_event_names
+https://github.com/opencivicdata/python-opencivicdata/zipball/master
```

Comment:
I hesitate to pin this to master, since (as we saw) this repo receives a fair amount of attention from its maintainers. 

You can pin the requirements to a particular commit, or wait for release 2.3 (the commit history indicates that 2.3 is coming very soon: https://github.com/opencivicdata/python-opencivicdata/commits/master).

Reply:
For this PR, can you pin it to the most recent commit, and then open an issue with a reminder to change the requirement to 2.3, when it comes out? (Just "watch" the ocd repo.)

Reply:
Done! The OCD line in `requirements.txt` is switched to a download URL I copied from Github. Let me know if there's a more elegant way to do this.